hadoop-plugin/src/main/groovy/com/linkedin/gradle/hadoopdsl/Workflow.groovy

```diff
@@ -364,6 +367,11 @@ class Workflow extends BaseNamedScopeContainer {
     flowDepends(clear, targetNames);
   }
 
+  @HadoopDslMethod
+  void conditions(String subflowCondition) {
```

Comment:
Should this be `conditions` or `condition`? If you're only able to specify a single condition then making this singular makes more sense to me

Reply:
Hmm, condition is more frequently used as a noun then the other examples you mentioned, but I see what you mean and it probably _can_ be used as a verb in this way.

I think `conditions` should be fine 👍 thanks!